tests/integ/test_transformer.py

```diff
@@ -297,6 +298,8 @@ def test_transform_byo_estimator(sagemaker_session, cpu_instance_type):
         model_desc = sagemaker_session.sagemaker_client.describe_model(
             ModelName=transformer.model_name
         )
+        assert model_desc["EnableNetworkIsolation"]
```

Comment:
[Food for thought] Is there a way to verify a wider range of variables are getting passed in properly using some sort of a dynamic check?

Reply:
I think we'd have to look for a struct that's identical to the output of `DescribeModel` for us to be able to do that dynamically. It wouldn't be technically hard to implement some kind of method to do that for Estimator/Model/etc. classes, but I'm not entirely sure how that would fit in. It does also begin to sound similar (but not the same) to what we have for workflows/Airflow.